inst/templates/wercker.yml

```diff
@@ -0,0 +1,5 @@
+box: rocker/hadleyverse
```

Comment:
This is a fair point, I would actually prefer to just use `rocker/r-base`, but it seems incapable of installing `devtools` because of a missing `curl` dependency.

You think it would be possible to get a working R on https://registry.hub.docker.com/_/busybox/.  2.5 Mb base image is pretty appealing :smile:.